doc/Extensions/Customizing-the-Web-UI.md

```diff
@@ -74,6 +79,11 @@ Now we can use that in the device menu entry to open winbox
     ```bash
     lnms config:set html.device.links.+ '{"url": "winbox://{{ $device->ip }}", "title": "Winbox"}'
     ```
+    
+    If your devices were added by IP address rather than hostname:
+    ```bash
+    lnms config:set html.device.links.+ '{"url": "winbox://{{ $device->hostname }}", "title": "Winbox"}'
```

Comment:
It should pretty much always be the second one (if you add by IP, the IP is in hostname)